src/mpd.c

```diff
@@ -2066,6 +2066,65 @@ mpd_get_query_params_find(int argc, char **argv, struct query_params *qp)
 }
 
 static int
+mpd_command_count(struct evbuffer *evbuf, int argc, char **argv, char **errmsg)
+{
+  struct query_params qp;
+  struct count_info ci;
+  int ret;
+
+  if (argc < 3 || ((argc - 1) % 2) != 0)
+    {
+      DPRINTF(E_LOG, L_MPD, "Missing argument(s) for command 'find'\n");
+      ret = asprintf(errmsg, "Missing argument(s) for command 'find'");
```

Comment:
You are right, no idea why i kept logging them twice (the caller "mpd_read_cb" already logs them). Added a commit that removes hopefully all them.